ui/admin/app/styles/app.scss

```diff
@@ -234,3 +234,13 @@
     }
   }
 }
+
+.choose-path {
+  display: block;
```

Comment:
could this be moved to the specific component? in this case, to `<Rose::Layout::Centered>`?

Reply:
This style is applied to `<Rose::Layout::Centered>`.